modules/BiomesAPI/build.gradle

```diff
@@ -69,11 +67,44 @@ for (dependency in moduleConfig.dependencies) {
 // Gradle uses the magic version variable when creating the jar name (unless explicitly set somewhere else I guess)
 version = moduleConfig.version
 
+import org.gradle.internal.logging.text.StyledTextOutputFactory
+import static org.gradle.internal.logging.text.StyledTextOutput.Style
+
+// The only case in which we make module non-snapshots is if release management is enabled and BRANCH_NAME is "master"
+if (moduleConfig.isReleaseManaged && env.BRANCH_NAME == "master") {
+    // This is mildly awkward since we need to bypass by default, yet if release management is on (true) then we set the bypass to false ..
+    ext.bypassModuleReleaseManagement = false
+    if (version.contains("-SNAPSHOT")) {
```

Comment:
```suggestion
    if (version.endsWith("-SNAPSHOT")) {
```

Reply:
Applied this, thanks! Actually cleaned it up a bit more yet now that I can brain again.